storybook-static/iframe.html

```diff
@@ -0,0 +1,74 @@
+<!doctype html><html lang="en"><head><meta charset="utf-8"><title>Storybook</title><meta name="viewport" content="width=device-width,initial-scale=1"><base target="_parent"><style>:not(.sb-show-main) > .sb-main,
```

Comment:
I commited the storybook static build as well, so we could base the github pages build on the main branch instead of gh-pages as proposed by @timbl in https://github.com/solid/solid-ui/issues/329

If we do not want to do that, I will remove the static files and add them to the gh-pages branch

Reply:
Might also be a case for checking out GitHub actions, to automatically build upon successful merges to master.

Reply:
I removed it, let's automate the build instead